tests/test_instance.py

```diff
@@ -13,6 +14,34 @@
 from conftest import CREATE_INSTANCE_METHOD, DELETE_INSTANCE_METHOD, GET_INSTANCE_METHOD
 
 
+@pytest.fixture(scope="session", autouse=True, params=[True, False])
+def configuration_mock(request):
```

Comment:
(big maybe, I'll let you see if this comment is applicable) This fixture currently depends on quite a lot of code to work properly: setting the path from the environment and deserializing the configuration. Instead I suggest that this fixture should directly build the `Configuration` object, and return (or yield) it. Any test that depends on the configuration, like `build_grpc_channel` later, or any other using an `Instance` probably should then explicitly use this fixture.

Reply:
Looks good, thanks!